modules/ROOT/pages/architecture/hexagonale_architecture.adoc

```diff
@@ -0,0 +1,218 @@
+:imagesdir: ../images
+// Open Questions/Discussion: 
+// - Is everything considered in the structure or are there things we cannot handle currently?
+// - What about shared technical elements? (AOP for logging across multiple rest adapters)
+// - How to organize multiple modules in a modular monolith and what about shared stuff there? 
+// - Should we really name the adapters after the protocol or more speaking (e.g. reservation for a inbound adapter for reservation) 
+// - Is structuring core and adapters in packages enough or should we use gradle modules or java modules?
+
+// Todos:
+// - Create ADR (Architecture Design records)
+// - Mention that the structure can be made more lean at any time. Consider three case:
```

Comment:
To me the most important decision is, is it sufficient to have one module with a good package structure or do I need more modules with specific dependencies.

Reply:
One more thing that is interesting in that context. What to do with multiple bounded contexts? Do we need two applications or can we put it inside one application? How to the two hexagons then communicate with each other.

Reply:
Created an issue for this #42